src/TextUI/TestRunner.php

```diff
@@ -470,34 +470,30 @@ public function doRun(Test $suite, array $arguments = [], array $warnings = [],
             if (isset($arguments['configuration'])) {
                 $filterConfiguration = $arguments['configuration']->getFilterConfiguration();
 
-                if (!empty($filterConfiguration['whitelist'])) {
-                    $whitelistFromConfigurationFile = true;
-                }
+                $whitelistFromConfigurationFile = true;
```

Comment:
In light of https://github.com/sebastianbergmann/phpunit/pull/4362/files#r449600526, the question is whether this variable makes sense.

What do you think, @sebastianbergmann?